packages/form-js-playground/src/components/PlaygroundRoot.js

```diff
@@ -91,14 +91,14 @@ export function PlaygroundRoot(props) {
   useEffect(() => {
     const dataEditor = dataEditorRef.current = new JSONEditor({
       value: toString(data),
-      contentAttributes: { 'aria-label': 'Form Input' },
+      contentAttributes: { 'aria-label': 'Form Input', tabIndex: 0 },
```

Comment:
isn't it better to use `undefined` as default?

Reply:
I think the creators of Code Mirror suggested a non-undefined value as the parent `.cm-scroller` container is `-1`. This is not configurable so we need to set the `cm-content` container explicitly.

![image](https://github.com/bpmn-io/form-js/assets/9433996/6dc24462-f72d-4bb5-930c-fdffa4263f98)

This is where the a11y error comes from in the first place

![image](https://github.com/bpmn-io/form-js/assets/9433996/92503248-aea4-4ea4-97ff-f9377c05a73a)